djangoProject/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.11.4-slim
```

Comment:
Pin the specific version to avoid accidental upgrades that may cause compatibility issues:
```
FROM python:3.11.4-slim@sha256:<specific-digest>
```